packages/disputer/package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "bin": "index.js",
   "scripts": {
-    "test": "hardhat test --network hardhat"
```

Comment:
for some reason having --network hardhat made ci fail, same as in the liquidator PR.

Reply:
I think it's an issue in our provider util. I've debugged it a bit but have not submitted a pr just yet.